scripts/upload-housing.js

```diff
@@ -22,9 +24,25 @@ const warnAndExit = (warning) => {
   process.exit(1);
 };
 
-const getJson = (jsonFile) => {
-  const rawdata = fs.readFileSync(jsonFile);
-  return JSON.parse(rawdata);
+const getCsv = (csvFile, field, docRef, db) => {
+  const rawdata = fs.readFileSync(csvFile, "utf-8");
+  parse(rawdata, { columns: true }, async (err, records) => {
+    try {
+      let dt = aq.from(records);
+      const batchCommits = [];
+      // var docRef = db.collection("housing_test").doc(csvFile);
+      let batch = db.batch();
+      let obj = {};
+      obj[field] = dt.objects();
+      batch.set(docRef, obj);
+      batchCommits.push(batch.commit());
+      return batchCommits;
+      // return Promise.all(batchCommits);
```

Comment:
Seems like you just want the data at this point instead of uploading to firebase at this moment?

```suggestion
      return dt = aq.from(records).objects();
```